website/docs/fun/anon-channel/main-cc.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Main CC
+---
+
+This command sends anonymous messages and maintains the sticky message with the anonymous message submit button.
+
+For more information about the anonymous channel, see [the overview](overview) page.
+
+## Trigger
+
+**Type:** `Regex`<br />
+**Trigger:** `\A`
+
+:::danger
+
+This command should be set to _only run in_ your anonymous channel in the channel restrictions.
+
+:::
+
+## Configuration
+
+See the [anonymous channel overview](overview/#configuration) for more information regarding how to configure this command.
```

Comment:
```suggestion
```
As there's no configuration necessary for this sub-CC, no need for this header; even less so if it just points to the main CC.

Reply:
Even though there is optional configuration?

Reply:
I must have missed that during review — in that case do ignore.